### PR TITLE
Final countdown

### PR DESCRIPTION
**Pull request by sudo-joseph**
Final tweaks for site. 
1. Fix static images not working in prod. 
2. Add on mouseover/mouseoff events for Cards. Corresponding map marker now grows when linked card is hovered over. 
3. replace react-modal with custom modal for geolocation failure modal.
4. Remove unused Fetch component.

### CODE REVIEW

File: client/src/App.js [hunk 1]
```diff
@@ -140,6 +150,18 @@ fetchFacilityDetails = (facility_id) => {
...
+  let url = `/api/facilities/earth911/coords/94608`
+  fetch(url,{ })
+    .then((response) => {
+      return response.json();
+    })
+    .then((location) => {
+      // this.setState({facilities:facilities_data.results.result});
+    console.log(location)
+    });
+}
```
Comment (ishinan) [inline client/src/App.js:163]: just a template?

File: client/src/App.js [hunk 2]
```diff
@@ -304,20 +329,21 @@ render() {
...
+              footer={<Button onClick={this.closeGeoLocationModal}>OK</Button>}>
+              <div>
+                  <div className="App-Modal-Content">
+                    <p>We are unable to detect your location. Please provide your zip code
+                    so that we can provide local results</p>
+                  <h3>Enter Zip:</h3>
+                  <Zip onValue={(value) => {this.enterZip(value)}}/>
+                  </div>
+              </div>
+             </Modal>
```
Comment (ishinan) [inline client/src/App.js:346]: Cool!

File: client/src/App.js [hunk 3]
```diff
@@ -339,7 +365,9 @@ render() {
                                     openModalHandler={this.openModalHandler}
                                     facilityModal={this.state.facilityModal}
                                     closeModalHandler={this.closeModalHandler}
-                                    facilityDetails={this.state.facilityDetails}/>
+                                    facilityDetails={this.state.facilityDetails}
+                                    onMouseOver={this.onMouseOver}
+                                    onMouseOut={this.onMouseOut}/>
```
Comment (ishinan) [inline client/src/App.js:370]: This is what I was thinking. I just thought we do not need to pass this from App.js, though.

Comment (ishinan) [reply client/src/App.js:370]: No need to change anything. This is just a comment.

File: server.js [hunk 4]
```diff
@@ -50,17 +50,14 @@ app.use(logger);
...
-  res.sendFile(path.join(__dirname, 'client/build/index.html'));
-});
-
-//The 404 Route (ALWAYS Keep this as the last route)
-app.get('*', function(req, res){
-  res.status(404).send('404');
+// For production, handle any requests that don't match the ones above
+app.use('/',express.static(path.join(__dirname, 'client/build')));
+app.use('/about/',express.static(path.join(__dirname, 'client/build')));
+app.use('/materials/',express.static(path.join(__dirname, 'client/build')));
```
Comment (ishinan) [inline server.js:56]: Looks like we needed some hack... okay as long as it is working :)

**Review by ishinan — APPROVED** [review]
Great work!

File: client/src/App.js (hunk 1 above)
Comment (sudo-joseph) [reply client/src/App.js:163]: API was not working for me :(

File: client/src/App.js (hunk 3 above)
Comment (sudo-joseph) [reply client/src/App.js:370]: I could have done this at the RecyclePage level but made more sense to me to keep all functions in App.js. Not sure what is correct process.

**Pull request merged**